extensions/amp-analytics/0.1/events.js

```diff
@@ -1468,6 +1472,29 @@ export class VisibilityTracker extends EventTracker {
       );
     }
 
+    const getUnlistenPromiseForSelector = (selector, selectionMethod) => {
```

Comment:
Make this a private method, so you don't need to initiate the function everytime.